gnomad_qc/v3/resources.py

```diff
@@ -1,131 +1,153 @@
 import hail as hl
-
+from gnomad_hail import logger
+from gnomad_hail.resources.resource_utils import (
+    MatrixTableResource,
+    TableResource,
+    PedigreeResource,
+    VersionedPedigreeResource
+)
+from gnomad_hail.resources.grch38 import na12878_giab
 V3_RELEASE_VERSION = 'r3.0'
 
-v3_qc_mt_path = 'gs://gnomad/sample_qc/mt/genomes_v3/gnomad_v3_qc_mt_v2_sites_dense.mt'
-v3_pc_relate_pca_scores_ht_path = 'gs://gnomad/sample_qc/mt/genomes_v3/gnomad_v3_qc_mt_v2_sites_pc_scores.ht' # TODO -> move to /ht
-v3_relatedness_ht_path = 'gs://gnomad/sample_qc/mt/genomes_v3/gnomad_v3_qc_mt_v2_sites_relatedness.ht' # TODO -> move to /ht
-v3_sex_ht_path = 'gs://gnomad/sample_qc/ht/genomes_v3/gnomad_v3_sex.ht'
+# MatrixTableResources
+v3_qc = MatrixTableResource('gs://gnomad/sample_qc/mt/genomes_v3/gnomad_v3_qc_mt_v2_sites_dense.mt')
+
+# TableResources
+gnomad_v3_genotypes = MatrixTableResource("gs://gnomad/raw/hail-0.2/mt/genomes_v3/gnomad_genomes_v3.repartitioned.mt")
```

Comment:
Not a big deal but this should probably move up to the MatrixTableResources section if you are going to have the comment separating them